data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/unapproachable_east/una_classes.lst

```diff
@@ -85,14 +113,38 @@ CLASS:Nentyar Hunter	SPELLSTAT:WIS	KNOWNSPELLS:LEVEL=1|LEVEL=2|LEVEL=3|LEVEL=4|L
 4	CAST:0,3,2,2,1																		BONUS:SKILL|Spot|4|TYPE=ClassAbility
 5	CAST:0,3,3,2,2,1
 
+
 # Class Name			Hit Dice	Type			Abbreviation	Source Page		Combat bonus											Bonus
-CLASS:Raumathari Battlemage	HD:4		TYPE:PC.Prestige	FACT:Abb|Rau		SOURCEPAGE:p.29	BONUS:COMBAT|BASEAB|classlevel("APPLIEDAS=NONEPIC")*3/4|TYPE=Base.REPLACE	BONUS:SAVE|BASE.Fortitude,BASE.Reflex|classlevel("APPLIEDAS=NONEPIC")/3	BONUS:SAVE|BASE.Will|classlevel("APPLIEDAS=NONEPIC")/2+2
+CLASS:Raumathari Battlemage	HD:4		TYPE:PC.Prestige	ABB:Rau		SOURCEPAGE:p.29	BONUS:COMBAT|BASEAB|classlevel("APPLIEDAS=NONEPIC")*3/4|TYPE=Base.REPLACE	BONUS:SAVE|BASE.Fortitude,BASE.Reflex|classlevel("APPLIEDAS=NONEPIC")/3	BONUS:SAVE|BASE.Will|classlevel("APPLIEDAS=NONEPIC")/2+2
```

Comment:
Master branch should be using latest available version. Currently 6.06.00RC2 the development autobuild.

The documentation is lacking the updates. The only source for FACT and FACTSET is here:
http://wiki.pcgen.org/FACT_Token
and
http://wiki.pcgen.org/FACTSET_Token
with additional information here:
http://wiki.pcgen.org/FreeMarker_Facet_Output

Best,